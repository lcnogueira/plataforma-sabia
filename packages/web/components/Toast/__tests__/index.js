import React from 'react';
import { render } from 'test-utils';
import { ToastContainer } from '..';

describe('ToastContainer component', () => {
	it('render correctly', () => {
		const { container } = render(<ToastContainer />);

		expect(container).toMatchSnapshot();
	});
});
