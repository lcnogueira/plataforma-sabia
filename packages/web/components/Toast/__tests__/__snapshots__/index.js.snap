// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastContainer component render correctly 1`] = `
@media only screen and (max-width:480px) {

}

<div>
  <div
    class="Toastify"
  />
</div>
`;
